image-layout.md

```diff
@@ -36,27 +36,46 @@ $ shasum -a 256 ./blobs/sha256-afff3924849e458c5ef237db5f89539274d5e609db5db935e
 afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51 ./blobs/sha256-afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51
 ```
 
-Object names in the refs and blobs MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", the hyphen `-`, the dot `.`, and the underscore `_`.
+Object names in the `refs` and `blobs` MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", the hyphen `-`, the dot `.`, and the underscore `_`.
```

Comment:
A few comments on this line:

* We should include the digits `0` through `9`.
* Aren't the blob filenames already completely specified?  `{algo}-{hex-digest}` with a defined list of algorithms cannot exceed this set unless we explicitly assign an algorithm an odd tag.